proto/hummock.proto

```diff
@@ -187,9 +201,10 @@ message CompactTask {
     ASSIGN_FAIL_CANCELED = 5;
     SEND_FAIL_CANCELED = 6;
     MANUAL_CANCELED = 7;
-    EXECUTE_FAILED = 8;
-    JOIN_HANDLE_FAILED = 9;
-    TRACK_SST_ID_FAILED = 10;
+    INVALIDGROUP_CANCELED = 8;
```

Comment:
Please use new index for new enum. We shouldn't change original indexes.

Reply:
> Please use new index for new enum. We shouldn't change original indexes.

I want to arrange `CANCELED` together and arrange `FAILED` together, respectively.